src/BuiltInTools/BrowserRefresh/WebSocketScriptInjection.js

```diff
@@ -99,6 +99,11 @@ setTimeout(function () {
 
   function applyBlazorDeltas(deltas) {
     deltas.forEach(d => window.Blazor._internal.applyHotReload(d.moduleId, d.metadataDelta, d.ilDelta));
+    fetch('_framework/blazor-hotreload', { method: 'post', headers: { 'content-type': 'application/json' }, body: JSON.stringify(deltas) })
+      .then(response => {
+        const etag = response.headers['etag'];
```

Comment:
Should we assert on the response code before storing the header in session storage?